source/index.rst

```diff
@@ -18,5 +18,6 @@ The Real-Time Working Group's mission is to advocate for and work on memory mana
    Subprojects/subprojects
    Related-Projects/related_projects
    Resources/resources
+   rtos/index
```

Comment:
@razr I think I added this by mistake. Could you remove the rtos folder?

Reply:
done